hudi-common/src/test/java/org/apache/hudi/common/util/TestFSUtils.java

```diff
@@ -129,6 +129,7 @@ public void testProcessFiles() throws Exception {
         .noneMatch(s -> s.contains(HoodieTableMetaClient.METAFOLDER_NAME)));
     // Check if only files are listed including hoodie.properties
     Assert.assertEquals("Collected=" + collected2, 5, collected2.size());
+    tmpFolder.delete();
```

Comment:
wont this delete it for other tests as well?

Reply:
wont